examples/stable/test/property/canister_methods/on_low_wasm_memory/src/index.ts

```diff
@@ -1,73 +1,55 @@
 import { call, IDL, onLowWasmMemory, query, update } from 'azle';
 
-export default class MemoryManagementCanister {
-    storedBytes: Uint8Array = new Uint8Array(0);
-    hasLowMemoryHandlerBeenCalled: boolean = false;
+export default class {
+    storedBytes: Uint8Array = new Uint8Array();
+    onLowWasmMemoryCalled: boolean = false;
 
     @onLowWasmMemory
     onLowWasmMemory(): void {
-        this.hasLowMemoryHandlerBeenCalled = true;
-        this.storedBytes = new Uint8Array(0);
+        this.onLowWasmMemoryCalled = true;
     }
 
     @query([], IDL.Bool)
-    wasLowMemoryHandlerCalled(): boolean {
-        return this.hasLowMemoryHandlerBeenCalled;
-    }
-
-    @update([], IDL.Vec(IDL.Nat8))
-    clearBytes(): Uint8Array {
-        this.storedBytes = new Uint8Array(0);
-        return new Uint8Array(this.storedBytes);
-    }
-
-    @query([], IDL.Nat32)
-    getStoredByteCount(): number {
-        return this.storedBytes.length;
+    getOnLowWasMemoryCalled(): boolean {
```

Comment:
This is spelled incorrectly, should be `getOnLowWasmMemoryCalled`

Reply:
done